pages/workspace/tracking-plan/properties.mdx

```diff
@@ -22,9 +22,68 @@ relevant for that specific event, when you analyze your data. You might be want
 
 Event properties are sent with the event in the <Link passHref href="/workspace/tracking-plan/events#a-namelog-eventa-log-event"><a>event call</a></Link>.
 
-### <a name="user-properties"></a> User Properties
+## <a name="group-properties"></a> Group properties
 
-User properties to describe the state of your user. It is information that is
+Group properties describe the state of a group you defined previously.
+
+Similar to User properties, this information is usually relevant in a larger scope than for exactly that event.
+
+> 💡 **When should I use Groups and Group Properties?**
+>
+> <Link passHref href="/data-design/groups">
+>   <a>
+>     Read more here about how Group Properties work and why they may be the
+>     right choice for you
+>   </a>
+> </Link>
+> .
+
+**Example**
+
+For B2B companies, products are used by multiple companies, each consisting of a number of users.
+In that case, you can maintain the state of each company with group properties, increase the count of employees when a new user joins a particular company for example.
+
+**Functionality**
+
+Group properties are updated with the event in the <Link passHref href="/workspace/tracking-plan/events#a-nameupdate-groups"><a>Update Groups</a></Link> action.
+
+### Managing Groups
+
+To be able to manage Group you will need to define them.
+
+#### Creating Group Types
+
+You can create Group Types either from the Properties screen or from the
+`Update Groups` action from the Event details.
+
+* From the Properties screen click `+ Add Group Type`
+* From the Update Groups action click `+ Add Group Type`
+
+and fill in name and description of your Group Type.
+
+> 💡 Tip: Each Group Type will need an ID for association (e.g. `Workspace`
+> : `Avo`). If you would like to strictly define possible ids for a type
+> (e.g. `Role` : [one of `Admin`, `Editor`, `Viewer`, etc.] you can choose
+> `Property Id should match one of` and define your possibilities.
```

Comment:
Woooooowsa! HUGE IF TRUUUUU 🚀

Reply:
I think we've decided to slice it out this time. Also not really clear if it is necessary at all, like @bragur mentioned it can be like the user id, which never appears in the Avo UI.